ynh-dev

```diff
@@ -145,6 +147,21 @@ elif [ "$1" = "run" ]; then
 
 
 #####################
+## Halt running VM ##
+#####################
+elif [ "$1" = "halt" ]; then
+    vm=$(vagrant global-status | head -3 | tail -1)
```

Comment:
This returns only the last run Vagrant Box, how can we be sure that is the good one ?